docs/2023-06-08-meeting-transcript.md

```diff
@@ -0,0 +1,881 @@
+
+Number 3, prior art. So I mentioend Forge has had some kind of signature-ish stuff for years, like, uh, literally, probably a decade, and no one uses it, and what can we learn from that? 
+
+And then, lastly, what sort of UI on the platforms or launchers can we use, if we hypothetically had a perfect like, if we had a signing system that everyone agreed on how would it work. Yeah. 
+
+Emi: And uh, before we invite speakers to speak on this, I do want to talk about something that CurseForge did say, which was the fact that it doesn't necessitate being a required process. There is value in having a trusted badge where this mod is signed, but still being able to download unsigned mods, just having this displayed to users in the way that makes sense and doesn't inspire unnecessary worry of the issue. 
+
+We do have someone who would like to speak on the topics, I am going to invite them now.  
+
+Starchild: Yeah, so I think one of the problems right now is we haven't exactly... decide what we want to prevent: do we want to like, use that as an additional way to authenticate when publishing mods, in which it's really quite simple, Modrinth and CurseForge can just associate specific key with user, and use that when checking new upload, that's quite simple. 
+
+Or do we also want to have some checks inside the loader, like for example, um, I am thinking for example, for large modpacks, they could have list of people that are allowed to sign mods that are used in the modpacks. So like if you download a mod for... or like, a malware rather than a mod, to your mod install, it would get immediately flaged.
+
+I think that's only two things that are really relevant, maybe someone else has other ideas, but I don't really see how we can work on something else that is actually productive and actually to do something.
+
+Emi: thank you for your thoughts on that. Uh, we have a member of the Fractureiser team that would like to say something on code signing. 
+
+Jasmine: Yeah. Hello, I'm Jasmine. 
+
+So uh, my thought on code signing, is the one from where I recently uploaded something to the Maven Central repository, which enforces code signing. 
+
+And that process was mostly able to be done through, like the build script itself; the most painful part of the process was getting the documentation on how to do it.  
+
+So I feel like if we're able to get something done, where it's part of the Gradle system itself, then it might reduce the ... yeah, like sort of the friction between like, um, enable to do the code signing, and uh, like, uploading onto mod distribution sites. 
+
+But that's something we are gonna have to work together with various of the modding projects to get it done for example mods and such. 
+
+Emi: I'd like to bring up some of these things that have been messaged in the meeting's channel, because there are people talking in text chant not speaking on voice, so that's not being recorded, and I'd like to relay it. 
```

Comment:
```suggestion
Emi: I'd like to bring up some of these things that have been messaged in the meeting's channel, because there are people talking in text chat not speaking on voice, so that's not being recorded, and I'd like to relay it. 
```